tools/itch5cacheaware.cpp

```diff
@@ -0,0 +1,147 @@
+#include <jb/itch5/compute_book_cache_aware.hpp>
```

Comment:
This is another file that should be removed from this PR and introduced in the next one.